src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -32,39 +36,40 @@
 (defn message-timestamp
   ([message]
    [message-timestamp message false])
-  ([{:keys [timestamp-str outgoing content outgoing-status]} justify-timestamp?]
-   [react/view (when justify-timestamp?
-                 {:align-self                       :flex-end
-                  :position                         :absolute
-                  :bottom                           9 ; 6 Bubble bottom, 3 message baseline
-                  (if (:rtl? content) :left :right) 0
-                  :flex-direction                   :row
-                  :align-items                      :flex-end})
-    (when (and outgoing justify-timestamp?)
-      [icons/icon (case outgoing-status
-                    :sending   :tiny-icons/tiny-pending
-                    :sent      :tiny-icons/tiny-sent
-                    :not-sent  :tiny-icons/tiny-warning
-                    :delivered :tiny-icons/tiny-delivered
-                    :tiny-icons/tiny-pending)
-       {:width               16
-        :height              12
-        :color               colors/white
-        :accessibility-label (name outgoing-status)}])
-    [react/text {:style (style/message-timestamp-text outgoing)}
-     timestamp-str]]))
+  ([{:keys [timestamp-str outgoing content outgoing-status pinned in-popover?]} justify-timestamp?]
+   (when-not in-popover?
```

Comment:
could you please add a comment on what this in-popover? about

Reply:
To keep track if message is being shown in a pin-limit-popover list and disable some props / adjust style. Added comment in the code also.